entrypoint.d/backend-startup.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+echo "hibernate.connection.url: jdbc:postgresql://${POSTGRES_HOST}:5432/${POSTGRES_DB}" > /app/config.properties
+echo "hibernate.connection.username: ${POSTGRES_USER}" >> /app/config.properties
+echo "hibernate.connection.password: ${POSTGRES_PASSWORD}" >> /app/config.properties
+echo "hibernate.connection.driver_class: org.postgresql.Driver" >> /app/config.properties
+echo "hibernate.dialect: org.hibernate.dialect.PostgreSQLDialect" >> /app/config.properties
+
+exec java -server -Xmx1G -XX:+UnlockExperimentalVMOptions -XX:+OptimizeStringConcat -XX:+UseStringDeduplication -XX:+UseCompressedOops -XX:+UseNUMA -XX:+UseG1GC -Xshare:on -jar /app/piped.jar
```

Comment:
```suggestion
/bin/sh /hotspot-entrypoint.sh
```

Might be nicer to re-use the [original entrypoint](https://github.com/TeamPiped/Piped-Backend/blob/3b1bef532b6548bdbdc34a570954af51db475a35/Dockerfile#L31) here